Jenkinsfile

```diff
@@ -233,7 +232,7 @@ def reactNativeIntegrationTests(targetPlatform) {
       sh 'adb uninstall io.realm.tests.reactnative || true' // '|| true' because the app might already not be installed
     } else if (targetPlatform == "ios") {
       dir('ios') {
-        sh 'pod install'
+        sh 'pod install --verbose'
```

Comment:
Do we need `--verbose`?

Reply:
Not really, but I don't think it'll affect performance and otherwise it's pretty limited in relevant output.